src/player.js

```diff
@@ -1654,31 +1652,76 @@ export default class Player extends FakeEventTarget {
   }
 
   /**
-   * Get the player states.
-   * @returns {Object} - The states of the player.
+   * Gets the player state types.
+   * @returns {StateTypes} - The state types of the player.
    * @public
    */
-  get State(): { [state: string]: string } {
-    return PlayerStates;
+  get State(): StateTypes {
+    return StateType;
   }
 
   /**
-   * Get the player tracks types.
-   * @returns {Object} - The tracks types of the player.
+   * Gets the player tracks types.
+   * @returns {TrackTypes} - The tracks types of the player.
    * @public
    */
-  get Track(): { [track: string]: string } {
-    return TrackTypes;
+  get Track(): TrackTypes {
+    return TrackType;
   }
 
   /**
-   * Get the player log level.
-   * @returns {Object} - The log levels of the player.
+   * Gets the player log level types.
+   * @returns {LogLevelTypes} - The log level types of the player.
    * @public
    */
-  get LogLevel(): { [level: string]: Object } {
+  get LogLevelType(): LogLevelTypes {
```

Comment:
maybe keep the convention with `LogLevel` and change the above to `getLogLevel` and get it out from `Enums` fold